src/components/Review.jsx

```diff
@@ -0,0 +1,46 @@
+import { useParams } from "react-router-dom";
+import { fetchReviews, fetchReviewComments } from "./api";
+import { useEffect, useState } from "react";
+
+const Review = () => {
+	const { review_id } = useParams();
+	const [review, setReview] = useState({});
+	const [comments, setComments] = useState([]);
+	const [loading, setLoading] = useState(false);
+	useEffect(() => {
+		setLoading(true);
+		Promise.all([fetchReviews(review_id), fetchReviewComments(review_id)]).then(
```

Comment:
Similarly here, for the functionality of the current ticket, we don't need to be fetching comments. The logic is good, but try to stick only to what is needed for the moment.